server/service/IndividualServicesService.js

```diff
@@ -357,27 +358,36 @@ exports.redirectInfoAboutApprovalStatusChanges = function (body, user, originato
       let applicationAddress = body["subscriber-address"];
       let applicationPort = body["subscriber-port"];
       let subscriberOperation = body["subscriber-operation"];
+      let applicationProtocol = body["subscriber-protocol"];
 
       /****************************************************************************************
        * Prepare logicalTerminatinPointConfigurationInput object to 
        * configure logical-termination-point
        ****************************************************************************************/
 
-      let operationList = [
-        subscriberOperation
+      let operationNamesByAttributes = new Map();
+      operationNamesByAttributes.set("subscriber-operation", subscriberOperation);
+
+      let tcpObjectList = [
+        {
+          "protocol" : applicationProtocol,
+          "address": applicationAddress,
+          "port": applicationPort
+        }
       ];
       let logicalTerminatinPointConfigurationInput = new LogicalTerminatinPointConfigurationInput(
         applicationName,
         releaseNumber,
-        applicationAddress,
-        applicationPort,
-        operationList
+        tcpObjectList,
+        operationServerName,
+        operationNamesByAttributes,
+        individualServicesOperationsMapping.individualServicesOperationsMapping
       );
       let logicalTerminationPointconfigurationStatus = await LogicalTerminationPointService.createOrUpdateApplicationInformationAsync(
```

Comment:
As per the specification , it is find the application of relevant application name and update the information . But , here we are calling createOrUpdateApplicationInformationAsync . can we once check whether the functionality is working as per specification.

Reply:
As discussed, check the above mentioned scenario